### ISSUE TITLE
How to enable teleportation?

### ISSUE THREAD

**Issue by medinags** [Rufus31415/Simple-WebXR-Unity#47]
Hello, 
Thank you for this very useful package.
I am new to WebGL, but I have some experience with MRTK.
I am trying to use teleportation in the Oculus browser. But I can't get it to work. 
In MixedRealityToolkit it is active. 

![image](https://user-images.githubusercontent.com/78471817/155034042-8e42a80d-ac82-486a-bf03-6e4cc68d9efd.png)

Do I have to do something else to activate it?
Merci!

**Comment by Rufus31415** [comment]
SimpleWebXR inputs support teleportation.

https://github.com/Rufus31415/Simple-WebXR-Unity/blob/f00c57aa2241d464e6d151afcf3347266beb03fc/SimpleWebXR-Demo/Assets/SimpleWebXR/Scripts/MRTK/SimpleWebXRController.cs#L29

https://github.com/Rufus31415/Simple-WebXR-Unity/blob/7b37e75ede96f70f4f16bb968c1b487c889ba782/SimpleWebXR-Demo/Assets/SimpleWebXR/Scripts/MRTK/SimpleWebXRHand.cs#L43

But it seems to me that you have to select a teleportation compatible pointer

**Comment by medinags** [comment]
Hello,
Thank you for replying
In inputs I have the SimpleWebXR profile
![image](https://user-images.githubusercontent.com/78471817/155286215-00f2de9c-7b34-4434-9402-403eda7865b8.png)
And I have this as a Pointer
![image](https://user-images.githubusercontent.com/78471817/155286156-8306e66c-1674-46ee-9395-9e6729f7f873.png)
Do I have to select another one?

**Comment by Rufus31415** [comment]
OK, everything seems ok, if you find what's going on, can you tell me ?

**Comment by medinags** [comment]
I'm testing the scene MRTK-HandInteraction

There is a script called "". That is supposed to execute the teleportation, but what it does is to move the player.
OculusQuestThumstickHandler.cs
https://github.com/Rufus31415/Simple-WebXR-Unity/blob/7b37e75ede96f70f4f16bb968c1b487c889ba782/SimpleWebXR-Demo/Assets/SimpleWebXR/Examples/Scripts/MRTK-HandInteraction/OculusQuestThumstickHandler.cs#L28

Another thing, did you test the teleportation in this scene, or should it be added from a new script?

**Comment by Rufus31415** [comment]
Yes this is just a workaround to move the player. But it doesn't use MRTK teleport feature.

Yes I tested teleportation, but I don't think there are any examples online that show this
